docs/sources/datasources/concepts.md

```diff
@@ -0,0 +1,150 @@
+
+## Plugins
+
+A plugin extends Grafana’s core functionality. Plugins can add new data source types, visualization panels, or full-featured applications that integrate with Grafana. They make Grafana modular and extensible.
+
+Plugins come in three types:
+
+- **Data source plugins** connect Grafana to **external data sources**. You use this type of plugin when you want to access and work with data from an external source or third party. Examples include Prometheus, MSSQL, and Databricks.
+
+- **Panel plugins** control how data appears in Grafana dashboards. Examples of panel plugins include pie chart, candlestick, and Plotly. 
+
+- **App plugins** allow you to bundle data sources and panel plugins within a single package. They also enable you to create custom pages within Grafana, providing a dedicated space for documentation, sign-up forms, and integration with other services via HTTP. Examples include Grafana Metrics Drilldown, Azure Cloud Native Monitoring, and Redis Application.
+
+## Integrations
+
+_Integrations are exclusive to Grafana Cloud._ An integration is a pre-packaged monitoring solution that bundles export/scrape configurations, pre-built dashboards, alert rules, and sometimes recording rules. Unlike standalone data sources, integrations handle the complete workflow: they configure how telemetry is collected and sent to Grafana Cloud's hosted databases, then provide ready-to-use dashboards and alerts. For example, a Kubernetes integration configures metric collection from your cluster, creates dashboards for monitoring, and sets up common alerts—all working together out of the box
+
+## When to use each
+
+Use a data source when:
+
+- You want to connect Grafana to a specific system (for example, Prometheus or MySQL).
+- You’re building custom dashboards with hand-picked metrics and visualizations.
+- Your monitoring needs are unique or not covered by pre-packaged integrations.
+
+Use a plugin when:
+
+- You need to connect to a system Grafana doesn’t support natively.
+- You want to add new functionality (visualizations, workflows, or app-style extensions).
+- You have specialized or industry-specific requirements (for example, IoT).
+
+Use an integration when:
+
+- You’re using Grafana Cloud and want a quick, pre-built setup.
+- You prefer minimal configuration with ready-to-use dashboards and alerts.
+- You’re new to observability and want to learn what good monitoring looks like.
```

Comment:
I think this still misses that data sources query the data from the underlying database, integrations are about ingesting the data into one of the hosted databases like mimir

Reply:
Should we not attempt to cover it here?